### PR TITLE
Add Access-Control-Allow-Origin header

### PR DESCRIPTION
**Pull request by RobLoach**

### CODE REVIEW

**Comment by soniktrooth** [comment]
@RobLoach do you think adding COR headers is going to jeopardize our chances with the module review process? Perhaps we should instead implement a hook so that other modules can add headers if they want.

**Comment by RobLoach** [comment]
This allows the CSS and JS assets to be consumed from a different domain.  Don't see it breaching any kind of security that would negatively impact reviews.

File: kalastatic.module [hunk 1]
```diff
@@ -176,6 +176,9 @@ function kalastatic_get_drupal_asset($type) {
 function kalastatic_asset_deliver($assets) {
   drupal_add_http_header('Content-Type', 'application/json; charset=utf-8');
```
Comment (RobLoach) [inline kalastatic.module]: @soniktrooth The other thing I noticed was that while we're outputting HTML here, its content type is set to JSON. Think we should remove this line?

**Comment by soniktrooth** [comment]
Ok, just wanted to check.

RE content type being application/json: without this we get an empty page with the style tags in the <head> where as what we actually want is the style tags as a string so it can be ingested. If this isn't want you want in your use case for EECS then you may have to do something custom. kalastatic_asset_deliver() needs to stay as it is otherwise Kalastatic metalsmith won't be able to pull in the goodness from the Drupes.

**Pull request merged**